### ISSUE TITLE
GPS: Move settings into own UAVO, Add GPS tab in ModuleSettings UI

### ISSUE THREAD

**Issue by tracernz** [d-ronin/dRonin#660]
Most of the settings aren't currently exposed in the UI. There are now enough to make it worth moving them into their own UAVO IMO and consequently free up a little RAM+Flash on F1.

**Comment by mlyle** [comment]
No more F1; objection to closing WONTFIX?

**Comment by tracernz** [comment]
I still think this is a good idea and make ModuleSettings less of a dogs breakfast. The UI is another issue.

**Comment by mlyle** [comment]
Yah, there's a lot of refactor of settings uavos possible, though it will break existing configurations (unless we explicitly build migration code for, which there's not a great framework).  At the same time, priorities, you know?  If we open issues for things we're not actively tripping over, we could have a pretty-close-to-infinite set of cleanup issues open.

I think the bug database is most useful when it encompasses things that we're likely to consider reasonably high priorities.

**Comment by tracernz** [comment]
Indeed. It's harder to keep track of what is actually done and what isn't though when things are closed/not-done. :wink:
